static/app/utils/performance/segmentExplorer/segmentExplorerQuery.tsx

```diff
@@ -72,6 +76,7 @@ export function getRequestFunction(_props: QueryProps) {
     const {eventView} = props;
     const apiPayload: FacetQuery = eventView.getEventsAPIPayload(props.location);
     apiPayload.aggregateColumn = aggregateColumn;
+    apiPayload.order = '-sumdelta';
```

Comment:
🤔 Feels like we should eventually have a better interface to do these kinds of things as we make more complex queries. We've been kinda just hacking on these kinds of modifications onto `EventView` for quite some time now.

Reply:
Yeah I kinda agree, but it depends on what we're up to with the api request. You could manage things like order entirely with the eventview before it arrives at the query, but we also have to be careful eventview doesn't become a kitchen sink for all api behaviors. 

That aside, this line is probably a bad example for converting though, since after cleanup I'll likely remove the ability to pass order and hardcode sumdelta as the order on the backend.